### ISSUE TITLE
2018/12/10 - 12/16

### ISSUE THREAD

**Issue by ampcpmgp** [ampcpmgp/weekly-report#48]
## 学習 📚
* [Able Programming | 深層学習とは？](https://www.youtube.com/watch?v=O3ohRBi5-Og&t=271s) をお風呂で閲覧。
  * 線形で分離できないニューロンは、非線形での分離が必要になり、その場合は複数のニューロン（深層ニューラルネットワーク）が必要な理由を理解した。
* [simple object detection](https://github.com/tensorflow/tfjs-examples/tree/master/simple-object-detection) の魔改造中。まずはelectronで動かしてチャートを表示するところまで完了。
  * 上記サンプルで1画像につき1物体しか検知が出来なく、別なやり方を考える必要があることがわかった。
* 損失値は以下の2種類があることを理解した。 [参考URL](http://aidiary.hatenablog.com/entry/20161109/1478696865)
  * loss（訓練データの損失）
  * val_loss（バリデーションセットの損失）

## 仕事 💼
* 人の書いたjQueryを直すお仕事。コンポーネント化を提案したが納期に間に合わないため断念。

## ライフハック 🏡
* 飲酒後の次の日に頭が働かない問題について、ビタミンＣ＋麦茶を摂取してみたが、多少の効果しか感じられなかった。お酒の頻度は多くても週1程度ではあるが、断酒することを検討する。
